extensions/wikia/ThemeDesigner/ThemeSettings.class.php

```diff
@@ -170,7 +170,12 @@ private function saveImage(
 			$file->upload( $temp_file->getPath(), '', '' );
 			$temp_file->delete( '' );
 
-			$settings["{$name}"] = $file->getURL();
+			// FIXME: XW-3596 - this is hack
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this "FIXME" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)